examples/GrpcCli/test.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env sh
+
+# Script that exercises all the commands of the CLI.
```

Comment:
It might make more sense to move these descriptions to a small README file as that is a more natural place to look for instructions on how to use a given example

Reply:
Sure I was just too lazy to write a readme. Will reconsider ;)

Reply:
added 🧧